VultisigApp/VultisigApp/Views/ImageView/AsyncImageView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AsyncImageView.swift
+//  VultisigApp
+//
+//  Created by Artur Guseinov on 30.05.2024.
+//
+
+import SwiftUI
+import Foundation
+
+struct AsyncImageView: View {
```

Comment:
I think you can just modify ImageView.swift

Reply:
Fixed, now we have only one component for images. All encapsulated.